firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-# Unreleased 
+# Unreleased
+
+# 21.4.2
+- [fixed] Remove Guava dependency from the SDK. This should eliminate crashes caused
+  by user App depending on a wrong flavor of Guava (#1125).
```

Comment:
I thought GRPC still depended on Guava and the problem still exists, just at a different layer?

Reply:
Changed to a better description. Rachel wants us to provide more information than `internal improvements`.